workflow-ui/core-android/src/main/java/com/squareup/workflow1/visual/AndroidViewFactory.kt

```diff
@@ -11,21 +11,24 @@ import com.squareup.workflow1.ui.ScreenViewFactory
 import com.squareup.workflow1.ui.ScreenViewFactoryFinder
 import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
 import com.squareup.workflow1.ui.show
-import com.squareup.workflow1.visual.AndroidViewFactoryKey.default
 import com.squareup.workflow1.visual.ContextOrContainer.AndroidContainer
 import com.squareup.workflow1.visual.ContextOrContainer.AndroidContext
 
+// TODO: Should R extend Screen? Seems likely. And perhaps a high order
+//  built in AndroidViewFactory for Screen could be responsible for keeping
+//  View.environmentOrNull working.
+
 @WorkflowUiExperimentalApi
 public typealias AndroidViewFactory<R> = VisualFactory<ContextOrContainer, R, View>
 
+@WorkflowUiExperimentalApi
 public sealed interface ContextOrContainer {
-  @JvmInline
-  public value class AndroidContext(public val context: Context) : ContextOrContainer
+  public class AndroidContext(public val context: Context) : ContextOrContainer
```

Comment:
The `value class` thing wasn't actually giving any benefit, let's not lie to ourselves.